File: Assignment_No-27/Question_01.c
/*
1)Accept a program which accept string from user and count number of capital characters.
input:Marvellous Multi OS
output:4
*/
#include<stdio.h>
int CountCapital(char *str)
{

	int iCount=0;
	while(*str!='\0')
	{

		if(*str>='A' && *str<='Z')
		{

			iCount++;
		}
		str++;
	}
    
    return iCount;

}
int main()
{

	char Arr[20];
	int iRet=0;
	printf("Enter a string:\n");
	scanf("%[^'\n']s",Arr);

	iRet=CountCapital(Arr);

	printf("%d",iRet);
	return 0;

}

File: Assignment_No-35/Question_01.java
*1)write a program which accept string form user and count number of captial Character
input:"Marvellous Multi OS"

output:4
*/

////////////////////////////////////////////////////////////////////////////////////////////////
import java.util.*;

class Demo
{
     int CountCaptial(String str)
	{

		int iCount=0;
		int i=0;
		for(i=0;i<str.length();i++)
		{
			if(Character.isUpperCase(str.charAt(i)))
			{
				iCount++;
			}
		}
		return iCount;

	}
   
}

class ass176
{
    public static void main(String  Arg[])
    {
        
        int iRet=0;
         
		    Scanner sobj=new Scanner(System.in);


	   	System.out.println("Enter the name of string:");
	   	String str=sobj.nextLine();
     

        Demo dobj=new Demo();
       iRet=dobj.CountCaptial(str);

       System.out.println("Captial character is:"+iRet);


}

}
